analytics/lib/counts.py

```diff
@@ -449,6 +449,15 @@ def do_pull_minutes_active(property: str, start_time: datetime, end_time: dateti
     ) usercount2
     ON
         usercount1.user_id = usercount2.user_id
+    JOIN (
```

Comment:
This issue shouldn't require changing any of the sql, I think?

Reply:
Hmm, just now I removed that code and the test is passing. Can you help me with the role of [this](https://github.com/zulip/zulip/blob/master/analytics/lib/counts.py#L443):
```
JOIN (
        SELECT realm_id, user_id
        FROM analytics_usercount
        WHERE
            property = '15day_actives::day' AND
            end_time = %%(time_end)s
    ) usercount2
    ON
        usercount1.user_id = usercount2.user_id
```

Reply:
I would just ignore all the SQL; this is one of the trickiest queries and CountStats in the file. Basically we're trying to get the AND of the two properties in the query (15day_actives::day and whatever comes in the block above this)